CMakeLists.txt

```
@@ -53,7 +57,8 @@ add_definitions(-DUI_EXTENSION_GIT_SHA="${UI_EXTENSION_GIT_SHA}")
 build_static_extension(${TARGET_NAME} ${EXTENSION_SOURCES})
 build_loadable_extension(${TARGET_NAME} " " ${EXTENSION_SOURCES})
 
-target_link_libraries(${EXTENSION_NAME} OpenSSL::SSL OpenSSL::Crypto)
+target_link_libraries(ui_loadable_extension ${OPENSSL_LIBRARIES})
+target_link_libraries(ui_extension ${OPENSSL_LIBRARIES})
```

This should be strictly equivalent to the previous version. Though this would be even better:
```suggestion
target_link_libraries(${EXTENSION_NAME} ${OPENSSL_LIBRARIES})
```

---

So I guess from the above that I misunderstood originally that the extension was not meant to be loaded after starting DuckDB. But, __it does work now__.

If I build the extension with this modification, and issue `../duckdb -unsigned` from the `build/release/extension` directory and then ``INSTALL extension/ui.duckdb_extension; LOAD ui;` and `CALL start_ui()`; next, I can use the UI and walk through the Dutch trains example.